src/nxt_process.h

```diff
@@ -26,6 +28,11 @@ typedef nxt_int_t (*nxt_process_start_t)(nxt_task_t *task, void *data);
 typedef nxt_int_t (*nxt_process_restart_t)(nxt_task_t *task, nxt_runtime_t *rt,
     nxt_process_init_t *init);
 
+typedef struct {
+    nxt_int_t        flags;
+    nxt_conf_value_t *uidmap;
```

Comment:
@VBart Is it okay to defer the parsing of uid/gid map fields from conf to the time of use? I see that all other conf fields are parsed in the `nxt_main_process.c`, just before creating the worker process. But in this case, if I parse it there then we will have to keep a dynamically allocated structure during worker lifetime, also deal with frees on errors/exit as well.

These fields are used here: https://github.com/nginx/unit/pull/289/files#diff-587ef968d773bbe167f664f3d842bb92R139
A buffer is allocated there, fields are parsed, written to the /proc/<pid>/uid_map file and then free'd there.

If you feel it's better to handle this different, let me know and I can improve.